python/src/aiconfig/eval/metrics.py

```diff
@@ -103,55 +73,58 @@ def __lt__(self, other: Any) -> bool:
         return self.pos - self.neg < other.pos - other.neg
 
 
-async def _get_sentiment_scores(output_datum: str) -> TextSentimentScores:
-    nltk.download("vader_lexicon", quiet=True)  # type: ignore
-    sid = SentimentIntensityAnalyzer()
-    mapping: dict[str, float] = sid.polarity_scores(output_datum)  # type: ignore
+class GetPolarityScores(Protocol):
+    @abstractmethod
+    def __call__(self, text: str) -> dict[str, float]:
+        pass
+
+
+def _get_nltk_polarity_scores(text: str, model: str) -> dict[str, float]:
+    nltk.download(model, quiet=True)  # type: ignore
```

Comment:
Does this get cached, I assume?

Reply:
maybe under the hood, but when my flight wifi cut out it was hanging, so Im not entirely sure. That's what motivated this PR. This is really there just as an example, dont think we need to manually cache. I just wanted it not to run for real during tests.